### ISSUE TITLE
Suggestion: Add allow-try to the rapidoc.blade.php

### ISSUE THREAD

**Issue by rabol** [knuckleswtf/scribe#777]
### Scribe version

4.28.0

### Your question

Super cool to see the 'external' configuration option

Rapidoc allow  one to disable the 'Try' and it could be done like this:

```
<!doctype html> <!-- Important: must specify -->
<html>
<head>
    <meta charset="utf-8"> <!-- Important: rapi-doc uses utf8 characters -->
    <script type="module" src="https://unpkg.com/rapidoc/dist/rapidoc-min.js"></script>
</head>
<body>

<rapi-doc spec-url="{!! $metadata['openapi_spec_url'] !!}"
          render-style="read"
          allow-try="{!! $metadata['try_it_out']['enabled'] == true ? 'true' : 'false'!!}"
> </rapi-doc>
</body>
</html>
```

I know that I can publish the views and do the change myself, but then I have to publish all resources and that seems a little overkill to add a singe thing  especially when the 'Try' is a config value

### Docs

- [X] I've checked [the docs](https://scribe.knuckles.wtf/laravel), the [troubleshooting guide](https://scribe.knuckles.wtf/laravel/troubleshooting), and [existing issues](https://github.com/knuckleswtf/scribe/issues?q=is%3Aissue+), but I didn't find a solution

**Comment by shalvah** [comment]
That makes sense. I'm sure there's ways to configure the external renderers , but I didn't have time to look into that. Could you send a PR for that?

**Comment by shalvah** [comment]
Fixed in 4.29.0. Also, you can now specify custom HTML attributes: https://scribe.knuckles.wtf/laravel/reference/config#external
